### DISCUSSION TITLE
PLR m model residuals not normally distributed

### DISCUSSION THREAD

**Discussion by RashidBakirov** [DoubleML/doubleml-for-py#274] [Q&A] [answered]
I am using Light GBM for both m and l models. When analysed the residuals of both, I found that the residuals of m are not normally distributed (see the image). l residuals are normal. Would this invalidate the results? I also am interested to learn how exactly is the p-value and the confidence intervals are calculated, if you could refer me to any documentation on this, it would be much appreciated.
![visualization (2)](https://github.com/user-attachments/assets/a795d46b-c46e-464e-891f-96c3760ac9ca)

**Comment by SvenKlaassen** [comment, marked as answer]
@RashidBakirov it should not matter if the residuals are not normally distributed. The p-values and CIs are based on asymptotic normality of the estimator (based on an CLT). You can take a look at the [DoubleML Documentation](https://docs.doubleml.org/dev/guide/se_confint.html) or the original [Double ML Paper](https://arxiv.org/abs/1608.00060) for more information on the CIs.
